Makefile

```diff
@@ -187,7 +190,7 @@ $(DEV_REQUIREMENTS_TIMESTAMP): $(REQUIREMENTS_TIMESTAMP) $(DEV_REQUIREMENTS)
 	@touch $(DEV_REQUIREMENTS_TIMESTAMP)
 
 
-$(DOCKER_BUILD_TIMESTAMP): $(TIMESTAMPS) $(PYTHON_FILES) $(CURRENT_DIR)/Dockerfile
+$(DOCKER_BUILD_TIMESTAMP): $(TIMESTAMPS) $(PROJECT_FILES) $(CURRENT_DIR)/Dockerfile
```

Comment:
All files under the project directory are copied inside the docker container, so if one of these files change, we need to rebuild the docker image. Before we had everything in container (.git, .gitignore, buildspec.yml, ...) now we only have the `project` directory content which contains python files plus the logging configuration (logging-cfg-*.yml). Thats why we need this `PROJECT_FILES` in the dep. Actually we should filter the python cache `__pycache__` files in `PROJECT_FILES` as well as in the docker file but this would complicate the makefile and docker file. I think it is not worth it to do this filtering. Hope this helped ? :-)